redwood-treehouse/src/commonMain/kotlin/app/cash/redwood/treehouse/OnBackPressedCallbackService.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.treehouse
+
+import app.cash.redwood.ui.OnBackPressedCallback
+import app.cash.zipline.ZiplineService
+import kotlin.native.ObjCName
+
+/** Redwood's [OnBackPressedCallback] but implementing [ZiplineService]. */
+@ObjCName("OnBackPressedCallbackService", exact = true)
+public interface OnBackPressedCallbackService : ZiplineService {
```

Comment:
Why doesn't this one extend `OnBackPressedCallback`?

Reply:
`OnBackPressedCallback` is an `abstract class`, so I can only extend it by making `OnBackPressedCallbackService`. Per the `ZiplineService` KDoc, `ZiplineService` should be implemented by interfaces (i.e., not abstract classes).